resources/mysqlbinlog/mysqlbinlog_test.go

```diff
@@ -0,0 +1,46 @@
+package mysqlbinlog
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/utils"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRunBinary tests whether mysqlbinlog can be started on the target platform
+// to check whether the lib extraction is correct.
+func TestRunBinary(t *testing.T) {
+	var tarName string
+	var version string
+
+	switch {
+	case runtime.GOOS == "darwin" && runtime.GOARCH == "arm64":
+		tarName = "mysqlbinlog-8.0.28-macos11-arm64.tar.gz"
+		version = "mysqlbinlog-8.0.28-macos11-arm64"
+	case runtime.GOOS == "linux" && runtime.GOARCH == "amd64":
+		tarName = "mysqlbinlog-8.0.28-linux-glibc-2.17-x86_64.tar.gz"
+		version = "mysqlbinlog-8.0.28-linux-glibc-2.17-x86_64"
+	}
```

Comment:
IMO it's better to deal with all the branches like
```go
if runtime.GOOS && runtime.GOARCH {
    ...
} else if {
    ...
} else {
  t.Logf("Unsupported combination of OS[%s] and ARCH[%s]", runtime.GOOS, runtime.GOARCH)
  return
}
```

Reply:
BTW, do we need to add this test to the GitHub Actions matrix? @d-bytebase  https://github.com/bytebase/bytebase/blob/main/.github/workflows/install-bb.yml

Reply:
> IMO it's better to deal with all the branches like
> 
> ```go
> if runtime.GOOS && runtime.GOARCH {
>     ...
> } else if {
>     ...
> } else {
>   t.Logf("Unsupported combination of OS[%s] and ARCH[%s]", runtime.GOOS, runtime.GOARCH)
>   return
> }
> ```

What about 
```go
switch{
   ... 
   t.Logf("Unsupported combination of OS[%s] and ARCH[%s]", runtime.GOOS, runtime.GOARCH)
}
```

Reply:
I'm OK with that, WDYT of the style? @d-bytebase

Reply:
BTW we should return immediately if the combination is not supported. @h3n4l